lib/ansible/plugins/connection/nspawn.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import os
+import pipes
+import subprocess
+import traceback
+import shlex
+
+from ansible import constants as C
+from ansible.errors import AnsibleError
+from ansible.plugins.connection import ConnectionBase
+from ansible.module_utils.basic import is_executable
+from ansible.utils.unicode import to_bytes
```

Comment:
The standard place to import this from is now:  from ansible.module_utils._text import to_bytes